cmd/updater/update.sh

```diff
@@ -664,10 +645,12 @@ if [ ${RESUME_INSTALL} -eq 0 ] && ! $DRYRUN; then
     # Note that the SCRIPTPATH we're passing in should be our binaries directory, which is what we expect to be
     # passed as the last argument (if any)
     echo "Starting the new update script to complete the installation..."
+    echo "copy the update script"
+    cp /home/ubuntu//go-algorand/cmd/updater/update.sh "${UPDATESRCDIR}/bin/${FILENAME}"
     exec "${UPDATESRCDIR}/bin/${FILENAME}" ${INSTALLOPT} -r -c ${CHANNEL} ${DATADIRSPEC} ${NOSTART} ${BINDIRSPEC} ${HOSTEDSPEC} ${GENESIS_NETWORK_DIR_SPEC} ${UNKNOWNARGS[@]}
 
     # If we're still here, exec failed.
-    fail_and_exit "Error executing the new update script - unable to continue"
+    # fail_and_exit "Error executing the new update script - unable to continue"
```

Comment:
Can this be removed, or should there be an error message at this point?

Reply:
I think this should be restored.

Reply:
Yes sorry, will restore. Thank you for catching it